pokete.py

```diff
@@ -464,10 +464,10 @@ def remove_item(self, item, amount=1):
         ARGS:
             item: Generic item name
             amount: Amount of items beeing removed"""
-        assert amount > 0, "Amounts have to be positive"
-        assert item in self.inv, f"Item {item} is not in the inventory"
+        assert amount > 0, "Amounts have to be positive."
+        assert item in self.inv, f"Item {item} is not in the inventory."
```

Comment:
Personal thing, but if this is to be changed, I would use an exclamation mark, as the program will "crash" if the assertion fails.

```suggestion
        assert item in self.inv, f"Item {item} is not in the inventory!"
```